Octokit.Tests/Reactive/ObservableIssuesClientTests.cs

```diff
@@ -117,9 +117,9 @@ public class TheGetAllForRepositoryMethod
                     && d["sort"] == "created"
                     && d["filter"] == "assigned"), Arg.Any<string>())
                 .Returns(Task.Factory.StartNew<IApiResponse<List<Issue>>>(() => firstPageResponse));
-            gitHubClient.Connection.Get<List<Issue>>(secondPageUrl, Arg.Any<Dictionary<string,string>>(), null)
+            gitHubClient.Connection.Get<List<Issue>>(secondPageUrl, Arg.Any<Dictionary<string,string>>(), Arg.Is<string>(s => s == "application/vnd.github.squirrel-girl-preview"))
```

Comment:
You can simplify `Arg.Is<string>(s => s == "application/vnd.github.squirrel-girl-preview")` to just `"application/vnd.github.squirrel-girl-preview"`